### ISSUE TITLE
Adjust tab color with One Dark UI theme?

### ISSUE THREAD

**Issue by plaxdan** [thomaslindstrom/no-caffeine-syntax#6]
The tabs remain white with the one-dark ui theme. There's some related information [here](https://discuss.atom.io/t/how-to-make-my-tabs-look-like-texteditor-tabs-in-the-new-one-themes/14995).

![image](https://cloud.githubusercontent.com/assets/382348/7048577/19ab2f44-ddd1-11e4-9bec-2fb926bcfb84.png)

**Comment by thomaslindstrom** [comment]
Hi, is this an issue with the `one-dark-ui` theme, or `no-caffeine`? `no-caffeine` is a syntax theme and shouldn't affect anything outside of the text editor.

**Comment by plaxdan** [comment]
Honestly I'm not sure, however the fact that one-dark-ui changes the tab for other themes but not this one (which I love btw) implies that it's the syntax theme and not the ui theme. 

I've not created an atom theme before, but looking through the one-dark-ui code the [`.tab data-type'"TextEditor"`](https://github.com/atom/one-dark-ui/blob/d1d7be2705c836d123963b936e9b4b90aa9aa519/styles/tabs.less#L54) style from one-dark-ui uses `@tab-background-color-active` to color the tab. Maybe this is something you could take advantage of?

**Comment by thomaslindstrom** [comment]
Thanks! I've looked around and I think it's because the `One Dark` theme tries to grab the text editor background color to put it in the tab. So I guess I've got to feed it to the `One Dark` theme somehow. I'll look into this pretty soon and get back to you here.

**Comment by and0m** [comment]
It took me sometime to figure a solution to this. 

I added the following lines to the stylesheet (Atom->Open your stylesheet) 
```
//Colour of active tab's border (left)
.tab-bar .tab::before{
  background:#c7b993;
}

//Colour of active tab's text colour (left)
.tab-bar .tab[data-type="TextEditor"].active{
  color:#c7b993;
}

//Active tab's background colour
.tab-bar .tab[data-type="TextEditor"]::after{
  background:#202022;
  border-bottom:none;
}
```

**Comment by thomaslindstrom** [comment]
Thanks, @and0m! I included this in `v0.15.1`.
